### PR TITLE
link{H__g} + ell, R__u

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -1103,9 +1103,9 @@ \subsection{Transaction Validation}
 
 %where $s[i]$ equals the root of the state trie immediately after the execution of the transaction $B_{\mathbf{T}}[i]$, and $g[i]$ the total gas used immediately after said transaction.
 
-The given \textbf{gasUsed} must correspond faithfully to the transactions listed: ${B_{H}}_{\mathrm{g}}$, the total gas used in the block, must be equal to the accumulated gas used according to the final transaction:
+The given \textbf{gasUsed} must correspond faithfully to the transactions listed: \hyperlink{H__g}{${B_H}_{\mathrm{g}}$}, the total gas used in the block, must be equal to the accumulated gas used according to the final transaction:
 \begin{equation}
-{B_{H}}_{\mathrm{g}} = \ell(\mathbf{R})_{\mathrm{u}}
+\hyperlink{H g}{${B_H}_{\mathrm{g}}$} = \hyperlink{ell}{\ell}\hyperlink{R__u}{(\mathbf{R})_{\mathrm{u}}}
```
Comment (pirapira) [inline Paper.tex]: Why did `$ ... $` appear?  It's already math mode around.

**Comment by jamesray1** [comment]
Fixed a merge conflict.

**Comment by pirapira** [comment]
The diff is empty.

**Comment by jamesray1** [comment]
#651

**Pull request closed without merging**